packages/plugins/users-permissions/admin/src/hooks/usePlugins/index.js

```diff
@@ -18,10 +17,11 @@ const usePlugins = (shouldFetchData = true) => {
       dispatch({
         type: 'GET_DATA',
       });
+      const fetchClient = getFetchClient();
```

Comment:
Can this be `useFetchClient()`, because it is inside a hook already?

Reply:
The problem here is the use of useCallback, the function is wrapped on that, I will refactor it in another branch where I will remove the useless useCallback, wdyt?

Reply:
Why don't you call use `useFetchHook` at the root of the `usePlugin` hook? Imo there is no need to call it inside `useCallback`.

Tbh: I'm not sure this `useCallback` is useless, but would be interesting to find out :)

Reply:
I have another ticket https://strapi-inc.atlassian.net/browse/DX-570 related to the cleanup of every useCallback encountered in the axios migration, I have modified based on your suggestion and I had to add a comment in the useCallback dependency to avoid adding fetchClient as a dependency. 
I created a different ticket for the cleanup of the useCallback also to help the testing phase of this ticket and the others related to the axios migration